src/print/print.ts

```diff
@@ -5,7 +5,7 @@ export default async function print(
   file: string,
   printer?: string,
   options?: string[]
-): Promise<void> {
```

Comment:
why have you removed the `Promise<void>`?

Reply:
It actually returns the output of exec so it's misleading. Can put it back but I think type errors will start popping up if you try to use the new function

Reply:
Adding an explicit return type